internal/tools/orchestrator/services/addon/addon.go

```diff
@@ -2633,13 +2633,32 @@ func (a *Addon) ListCustomAddon() (*[]map[string]interface{}, error) {
 	basic := locale.Get("basicPlan")
 	professional := locale.Get("professionalPlan")
 
-	createableAddons := []string{"api-gateway", "mysql", "canal", "monitor"}
-	createableAddonVersion := map[string]string{"api-gateway": "3.0.0", "mysql": "5.7.29", "canal": "1.1.0", "monitor": "3.6"}
+	createableAddons := []string{
+		apistructs.AddonApiGateway,
+		apistructs.AddonMySQL,
+		apistructs.AddonMonitor,
+		apistructs.AddonRegisterCenter,
+		apistructs.AddonNewConfigCenter,
+	}
+	// if these addon-type is `custom` and they can't be deployed by user, skip it!
+	unCreateAbleAddons := []string{
+		apistructs.AddonMSENacos, // mse-nacos only can be deployed by `config-center` or `register-center`
+	}
+	createableAddonVersion := map[string]string{
+		apistructs.AddonApiGateway:      "3.0.0",
```

Comment:
these default `addon:version` pairs should be putted in envs or db configs, for easier modifying.

Reply:
oh yes, I found these version just for getting AddonSpec, now I've changed it to get AddonSpec from cache.